src/main.py

```diff
@@ -0,0 +1,63 @@
+import sys
+from github import Github
+import os
+
+
+def format_feedback(flake8_report):
+    flake8_report = open(flake8_report, 'r')
+    lines = flake8_report.readlines()
+
+    files = {}
+    count = 0
+    for line in lines:
+        content = line.strip().split(":", 3)
+        if len(content) < 4:
+            continue
+
+        filename = content[0]
+        if filename not in files:
+            files[filename] = []
+        files[filename].append({
+            'line': content[1],
+            'message': content[-1]
+        })
+        count += 1
+
+    return {
+        'count': count,
+        'files': files,
+    }
+
+
+def build_comment(feedback):
+    if feedback['count'] == 0:
+        return '### Nenhum erro foi encontrado.\n'
+
+    msg = 'Foram encontrados {} erros'.format(feedback['count'])
+    if feedback['count'] == 1:
+        msg = 'Foi encontrado {} erro'.format(feedback['count'])
```

Comment:
Já não podia fazer asim de uma vez?

```suggestion
        msg = 'Foi encontrado 1 erro'
```